<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1"><!-- Begin Jekyll SEO tag v2.4.0 -->
<title>Publications2 | Matthew R. Olm</title>
<meta name="generator" content="Jekyll v3.7.3" />
<meta property="og:title" content="Publications2" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Matt Olm - PhD Candidate in Jill Banfield’s lab at UC-Berkeley." />
<meta property="og:description" content="Matt Olm - PhD Candidate in Jill Banfield’s lab at UC-Berkeley." />
<link rel="canonical" href="http://localhost:4000/publications2/" />
<meta property="og:url" content="http://localhost:4000/publications2/" />
<meta property="og:site_name" content="Matthew R. Olm" />
<script type="application/ld+json">
{"@type":"WebPage","url":"http://localhost:4000/publications2/","description":"Matt Olm - PhD Candidate in Jill Banfield’s lab at UC-Berkeley.","headline":"Publications2","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<link rel="stylesheet" href="/assets/main.css"><link type="application/atom+xml" rel="alternate" href="http://localhost:4000/feed.xml" title="Matthew R. Olm" /></head>
<body><header class="site-header" role="banner">

  <div class="wrapper"><a class="site-title" rel="author" href="/">Matthew R. Olm</a><nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
              <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
              <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger"><a class="page-link" href="/CV/">Online CV</a><a class="page-link" href="/">About</a><a class="page-link" href="/publications/">Publications</a><a class="page-link" href="/publications2/">Publications2</a></div>
      </nav></div>
</header>
<main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post h-entry" itemscope itemtype="http://schema.org/BlogPosting">

  <header class="post-header">
    <h1 class="post-title p-name" itemprop="name headline">Publications2</h1>
    <p class="post-meta">
      <time class="dt-published" datetime="" itemprop="datePublished">
      </time></p>
  </header>

  <div class="post-content e-content" itemprop="articleBody">
    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
<p>
<body> <i><big><a href="https://www.biorxiv.org/content/10.1101/558676v1.article-metrics"> Necrotizing enterocolitis is preceded by increased gut bacterial replication, Klebsiella, and fimbriae-encoding bacteria that may stimulate TLR4 receptors</a></i> - <b>bioRxiv</b> (2019)</big><br/><b>Olm M.R.</b>,  Bhattacharya N.,  Crits-Christoph A.,  Firek B.A.,  Baker R.,  Song Y.S.,  Morowitz M.J.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1101/558676"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1101/558676 data-style="small_circle"></span>
</span>

</p>

<hr/>
<p>
<body> <i><big><a href="https://doi.org/10.1186/s40168-019-0638-1"> Genome-resolved metagenomics of eukaryotic populations during early colonization of premature infants and in hospital rooms</a></i> - <b>Microbiome</b> (2019)</big><br/><b>Olm M.R.</b>,  West P.T.,  Brooks B.,  Firek B.A.,  Baker R.,  Morowitz M.J.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1186/s40168-019-0638-1"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1186/s40168-019-0638-1 data-style="small_circle"></span>
</span>

</p>

<hr/>
<p>
<body> <i><big><a href="http://dx.doi.org/10.1038/s41564-018-0338-9"> Megaphages infect Prevotella and variants are widespread in gut microbiomes</a></i> - <b>Nature microbiology</b> (2019)</big><br/>Devoto A.E.,  Santini J.M.,  <b>Olm M.R.</b>,  Anantharaman K.,  Munk P.,  Tung J.,  Archie E.A.,  Turnbaugh P.J.,  Seed K.D.,  Blekhman R.,  Aarestrup F.M.,  Thomas B.C.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s41564-018-0338-9"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1038/s41564-018-0338-9 data-style="small_circle"></span>
</span>

</p>

<hr/>
<p>
<body> <i><big><a href="http://dx.doi.org/10.1038/s41467-018-08246-y"> Hydrogen-based metabolism as an ancestral trait in lineages sibling to the Cyanobacteria</a></i> - <b>Nature communications</b> (2019)</big><br/>Matheus Carnevali P.B.,  Schulz F.,  Castelle C.J.,  Kantor R.S.,  Shih P.M.,  Sharon I.,  Santini J.M.,  <b>Olm M.R.</b>,  Amano Y.,  Thomas B.C.,  Anantharaman K.,  Burstein D.,  Becraft E.D.,  Stepanauskas R.,  Woyke T.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s41467-018-08246-y"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1038/s41467-018-08246-y data-style="small_circle"></span>
</span>

</p>

<hr/>
<p>
<body> <i><big><a href="https://www.biorxiv.org/content/10.1101/530139v1.abstract"> Functional potential of bacterial strains in the premature infant gut microbiome is associated with gestational age</a></i> - <b>bioRxiv</b> (2019)</big><br/>Rahman S.F.,  <b>Olm M.R.</b>,  Morowitz M.J.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1101/530139"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1101/530139 data-style="small_circle"></span>
</span>

</p>

<hr/>
<p>
<body> <i><big><a href="https://www.biorxiv.org/content/early/2018/11/09/466417.abstract"> Shifts in microbial transcription patterns prior to onset of Necrotizing enterocolitis may indicate low oxygen levels in the gut</a></i> - <b>bioRxiv</b> (2018)</big><br/>Sher Y.,  <b>Olm M.R.</b>,  Raveh-Sadka T.,  Brown C.T.,  Sher R.,  Firek B.,  Baker R.,  Morowitz M.,  Banfield J. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1101/466417"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1101/466417 data-style="small_circle"></span>
</span>

</p>

<hr/>
<p>
<body> <i><big><a href="http://dx.doi.org/10.1186/s40168-018-0493-5"> The developing premature infant gut microbiome is a major factor shaping the microbiome of neonatal intensive care unit rooms</a></i> - <b>Microbiome</b> (2018)</big><br/>Brooks B.,  <b>Olm M.R.</b>,  Firek B.A.,  Baker R.,  Geller-McGrath D.,  Reimer S.R.,  Soenjoyo K.R.,  Yip J.S.,  Dahan D.,  Thomas B.C.,  Morowitz M.J.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1186/s40168-018-0493-5"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1186/s40168-018-0493-5 data-style="small_circle"></span>
</span>

</p>

<hr/>
<p>
<body> <i><big><a href="http://dx.doi.org/10.1128/mBio.00441-18"> Hospitalized Premature Infants Are Colonized by Related Bacterial Strains with Distinct Proteomic Profiles</a></i> - <b>mBio</b> (2018)</big><br/>Brown C.T.,  Xiong W.,  <b>Olm M.R.</b>,  Thomas B.C.,  Baker R.,  Firek B.,  Morowitz M.J.,  Hettich R.L.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1128/mBio.00441-18"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1128/mBio.00441-18 data-style="small_circle"></span>
</span>

</p>

<hr/>
<p>
<body> <i><big><a href="http://dx.doi.org/10.1128/mSystems.00123-17"> Machine Learning Leveraging Genomes from Metagenomes Identifies Influential Antibiotic Resistance Genes in the Infant Gut Microbiome</a></i> - <b>mSystems</b> (2018)</big><br/>Rahman S.F.,  <b>Olm M.R.</b>,  Morowitz M.J.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1128/mSystems.00123-17"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1128/mSystems.00123-17 data-style="small_circle"></span>
</span>

</p>

<hr/>
<p>
<body> <i><big><a href="http://biorxiv.org/content/early/2018/04/03/294421.abstract"> Microbial diversity and metabolic potential in cyanotoxin producing cyanobacterial mats throughout a river network</a></i> - <b>bioRxiv</b> (2018)</big><br/>Bouma-Gregson K.,  <b>Olm M.R.</b>,  Probst A.J.,  Anantharaman K.,  Power M.E.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1101/294421"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1101/294421 data-style="small_circle"></span>
</span>

</p>

<hr/>
<p>
<body> <i><big><a href="http://dx.doi.org/10.1038/ismej.2017.126"> dRep: a tool for fast and accurate genomic comparisons that enables improved genome recovery from metagenomes through de-replication</a></i> - <b>The ISME journal</b> (2017)</big><br/><b>Olm M.R.</b>,  Brown C.T.,  Brooks B.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/ismej.2017.126"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1038/ismej.2017.126 data-style="small_circle"></span>
</span>

</p>

<hr/>
<p>
<body> <i><big><a href="http://dx.doi.org/10.1038/s41467-017-02018-w"> Strain-resolved analysis of hospital rooms and infants reveals overlap between the human and room microbiome</a></i> - <b>Nature communications</b> (2017)</big><br/>Brooks B.,  <b>Olm M.R.</b>,  Firek B.A.,  Baker R.,  Thomas B.C.,  Morowitz M.J.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s41467-017-02018-w"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1038/s41467-017-02018-w data-style="small_circle"></span>
</span>

</p>

<hr/>
<p>
<body> <i><big><a href="http://dx.doi.org/10.1101/gr.213256.116"> Identical bacterial populations colonize premature infant gut, skin, and oral microbiomes and exhibit different in situ growth rates</a></i> - <b>Genome research</b> (2017)</big><br/><b>Olm M.R.</b>,  Brown C.T.,  Brooks B.,  Firek B.,  Baker R.,  Burstein D.,  Soenjoyo K.,  Thomas B.C.,  Morowitz M.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1101/gr.213256.116"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1101/gr.213256.116 data-style="small_circle"></span>
</span>

</p>

<hr/>
<p>
<body> <i><big><a href="http://dx.doi.org/10.1128/mBio.01969-16"> The Source and Evolutionary History of a Microbial Contaminant Identified Through Soil Metagenomic Analysis</a></i> - <b>mBio</b> (2017)</big><br/><b>Olm M.R.</b>,  Butterfield C.N.,  Copeland A.,  Boles T.C.,  Thomas B.C.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1128/mBio.01969-16"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1128/mBio.01969-16 data-style="small_circle"></span>
</span>

</p>

<hr/>
<p>
<body> <i><big><a href="http://dx.doi.org/10.1038/nbt.3704"> Measurement of bacterial replication rates in microbial communities</a></i> - <b>Nature biotechnology</b> (2016)</big><br/>Brown C.T.,  <b>Olm M.R.</b>,  Thomas B.C.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/nbt.3704"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1038/nbt.3704 data-style="small_circle"></span>
</span>

</p>

<hr/>
<p>
<body> <i><big><a href="http://dx.doi.org/10.1111/mmi.13414"> Function, expression, specificity, diversity and incompatibility of actinobacteriophage parABS systems</a></i> - <b>Molecular microbiology</b> (2016)</big><br/>Dedrick R.M.,  Mavrich T.N.,  Ng W.L.,  Cervantes Reyes J.C.,  <b>Olm M.R.</b>,  Rush R.E.,  Jacobs-Sera D.,  Russell D.A.,  Hatfull G.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1111/mmi.13414"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1111/mmi.13414 data-style="small_circle"></span>
</span>

</p>

<hr/>
<p>
<body> <i><big><a href="http://dx.doi.org/10.1128/JVI.03363-13"> Cluster M mycobacteriophages Bongo, PegLeg, and Rey with unusually large repertoires of tRNA isotypes</a></i> - <b>Journal of virology</b> (2014)</big><br/>Pope W.H.,  Anders K.R.,  Baird M.,  Bowman C.A.,  Boyle M.M.,  Broussard G.W.,  Chow T.,  Clase K.L.,  Cooper S.,  Cornely K.A.,  DeJong R.J.,  Delesalle V.A.,  Deng L.,  Dunbar D.,  Edgington N.P.,  Ferreira C.M.,  Weston Hafer K.,  Hartzog G.A.,  Hatherill J.R.,  Hughes L.E.,  Ipapo K.,  Krukonis G.P.,  Meier C.G.,  Monti D.L.,  <b>Olm M.R.</b>,  Page S.T.,  Peebles C.L.,  Rinehart C.A.,  Rubin M.R.,  Russell D.A.,  Sanders E.R.,  Schoer M.,  Shaffer C.D.,  Wherley J.,  Vazquez E.,  Yuan H.,  Zhang D.,  Cresawn S.G.,  Jacobs-Sera D.,  Hendrix R.W.,  Hatfull G.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1128/JVI.03363-13"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1128/JVI.03363-13 data-style="small_circle"></span>
</span>

</p>

<hr/>

  </div><a class="u-url" href="/publications2/" hidden></a>
</article>

      </div>
    </main><footer class="site-footer h-card">
  <data class="u-url" href="/"></data>

  <div class="wrapper">

    <h2 class="footer-heading">Matthew R. Olm</h2>

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li class="p-name">Matthew R. Olm</li><li><a class="u-email" href="mailto:mattolm@berkeley.edu">mattolm@berkeley.edu</a></li></ul>
      </div>

      <div class="footer-col footer-col-2"><ul class="social-media-list"><li><a href="https://github.com/mrolm"><svg class="svg-icon"><use xlink:href="/assets/minima-social-icons.svg#github"></use></svg> <span class="username">mrolm</span></a></li><li><a href="https://www.twitter.com/mattagenolmics"><svg class="svg-icon"><use xlink:href="/assets/minima-social-icons.svg#twitter"></use></svg> <span class="username">mattagenolmics</span></a></li></ul>
</div>

      <div class="footer-col footer-col-3">
        <p>Matt Olm - PhD Candidate in Jill Banfield&#39;s lab at UC-Berkeley.
</p>
      </div>
    </div>

  </div>

</footer>
</body>

</html>
