---
layout: page
title: Online CV
permalink: /CV/
---
<p align="center">
  <b><big><big>Matthew R. Olm</big></big></b> <br>
  mattolm@berkeley.edu
</p>

### Education

2014-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;PhD, Microbiology; The University of California, Berkeley  
2010-2014
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;BSc, Microbiology; Minor, Computer Science; The University of Pittsburgh  

### Fellowships and honors

2017 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*Genome Research* Cover Image  ([April](https://genome.cshlp.org/content/27/4.cover-expansion))  
2017&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*mBio* Editors' pick ([volume 8, issue 1](http://mbio.asm.org/site/misc/mBioCollection2017.xhtml))  
2014 - 2019 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;National Science Foundation Graduate Research Fellowship  
2012 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Howard Hughes Medical Institute Summer Research Fellowship

### Software

 **dRep** - Python program for rapid comparison and derepliaction of microbial genomes  
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Code](https://github.com/MrOlm/drep) \| [Publication](https://www.nature.com/articles/ismej2017126)  

 **iRep** - Estimating bacteria replication rates based on genome coverage variation  
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Code](https://github.com/christophertbrown/iRep) | [Publication](http://www.nature.com/nbt/journal/v34/n12/abs/nbt.3704.html)  

 *Other programs and scripts* - ([GitHub Account](https://github.com/MrOlm))

### Select scientific presentations

**Academic**
 * 2018 (poster) **Stanford Microbiome Symposium 2018**; Stanford, CA. *Metagenomic analysis reveals bacterial replication rates increase two days prior to necrotizing enterocolitis diagnosis*
 * 2018 (poster) **17th International Symposium on Microbial Ecology**; Leipzig, Germany. *De novo assembly of eukaryotic genomes from metagenomes reveals strain-level overlap between infant and hospital fungal microbiomes*
 * 2017 (oral presentation) **MMG Seminar Series at the University of Pittsburgh**; Pittsburgh, PA. *Using genome-resolved metagenomics to study the developing infant microbiome*
 * 2017 (poster) **Applied Bioinformatics & Public Health Microbiology**; Cambridge, UK. *Genome-resolved metagenomics of infant fecal samples yields thousands of high-quality microbial genomes and reveals patterns of sub-species diversity*
 * 2016 (oral presentation) **Lake Arrowhead Microbial Genomics Conference**; Lake Arrowhead, CA. *Identical Bacterial Populations Colonize Premature Infant Gut, Skin, and Oral Microbiomes and Exhibit Different in situ Growth Rates*
 * 2013 (poster) **American Society for Microbiology General Meeting**; Denver, CO. *Partitioning System of Bacteriophage RedRock Increases Plasmid Maintenance in Mycobacterium smegmatis.*
 * 2012 (poster) **Science 2012**; Pittsburgh, PA. *Study of Putative Partitioning System in Mycobacteriophage RedRock*


**Blog posts**
* [Why genome completeness and contamination estimates are more complicated than you think](https://www.microbe.net/2017/12/13/why-genome-completeness-and-contamination-estimates-are-more-complicated-than-you-think/) _microBE.net_, December 13, 2017  
* [Are these microbes the “same”?](https://www.microbe.net/2017/02/15/are-these-microbes-the-same/) _microBE.net_, February 15, 2017  


**Public engagement** (California Academy of Sciences NightLife; [description](https://www.calacademy.org/nightlife))
* *What's Living in my food? The Good/Bad stuff* (November 17, 2016)
* *History of Beer Microbiology* (July 7, 2016)
* *Who's living in my food? Peeking in on helpful and harmful food microbes* (November 19, 2015)
* *Microbial Fermentation - When, How, and Why?* (September 3, 2015)

### First and Second Author Publications

See [publications](/publications) for full list of publications


__Olm, Matthew R__\*, Patrick T West\*, Brandon Brooks, Brian A. Firek, Robyn Baker, Michael J. Morowitz, and Jillian F. Banfield. “Genome-Resolved Metagenomics of Eukaryotic Populations during Early Colonization of Premature Infants and in Hospital Rooms.” *Microbiome* 7 (1): 26 (February 15, 2019). <https://doi.org/10.1186/s40168-019-0638-1>

Rahman, Sumayah F., **Matthew R. Olm**, Michael J. Morowitz, and Jillian F. Banfield. 2019. “Functional Potential of Bacterial Strains in the Premature Infant Gut Microbiome Is Associated with Gestational Age.” *bioRxiv* (January 24, 2019). <https://doi.org/10.1101/530139>

Sher, Yonatan, **Matthew R Olm**, Tali Raveh-Sadka, Christopher T Brown, Ruth Sher, Brian Firek, Robyn Baker, Michael Morowitz, Jill Banfield. "Shifts in microbial transcription patterns prior to onset of Necrotizing enterocolitis may indicate low oxygen levels in the gut." *BioRxiv* (November 9, 2018). <https://doi.org/10.1101/466417>

Brooks, Brandon, **Matthew R Olm**, Brian A. Firek, Robyn Baker, David Geller-McGrath, Sophia R. Reimer, Karina R. Soenjoyo, et al. “The Developing Premature Infant Gut Microbiome Is a Major Factor Shaping the Microbiome of Neonatal Intensive Care Unit Rooms.” *Microbiome* 6:112 (June 20, 2018). <https://doi.org/10.1186/s40168-018-0493-5>.

Bouma-Gregson, Keith, **Matthew R Olm**, Alexander J Probst, Karthik Anantharaman, Mary E Power, and Jillian F Banfield. “Microbial Diversity and Metabolic Potential in Cyanotoxin Producing Cyanobacterial Mats throughout a River Network.” *BioRxiv* (In press at *The ISME Journal*) (April 3, 2018). <https://doi.org/10.1101/294421>.

Rahman, Sumayah F., **Matthew R. Olm**, Michael J. Morowitz, and Jillian F. Banfield. “Machine Learning Leveraging Genomes from Metagenomes Identifies Influential Antibiotic Resistance Genes in the Infant Gut Microbiome.” *MSystems* 3, no. 1 (February 27, 2018). <https://doi.org/10.1128/mSystems.00123-17>.

Brooks, Brandon, **Matthew R. Olm**, Brian A. Firek, Robyn Baker, Brian C. Thomas, Michael J. Morowitz, and Jillian F. Banfield. “Strain-Resolved Analysis of Hospital Rooms and Infants Reveals Overlap between the Human and Room Microbiome.” *Nature Communications* 8, no. 1 (November 27, 2017): 1814. <https://doi.org/10.1038/s41467-017-02018-w>.

**Olm, Matthew R**, Christopher T Brown, Brandon Brooks, and Jillian F Banfield. “DRep: A Tool for Fast and Accurate Genomic Comparisons That Enables Improved Genome Recovery from Metagenomes through de-Replication.” *The ISME Journal* 11 (July 25, 2017): 2864. <http://dx.doi.org/10.1038/ismej.2017.126>.

**Olm, Matthew R.**, Christopher T. Brown, Brandon Brooks, Brian Firek, Robyn Baker, David Burstein, Karina Soenjoyo, Brian C. Thomas, Michael Morowitz, and Jillian F. Banfield. “Identical Bacterial Populations Colonize Premature Infant Gut, Skin, and Oral Microbiomes and Exhibit Different in Situ Growth Rates.” *Genome Research* 27, no. 4 (April 1, 2017): 601–12. <https://doi.org/10.1101/gr.213256.116>.

**Olm, Matthew R.**, Cristina N. Butterfield, Alex Copeland, T. Christian Boles, Brian C. Thomas, and Jillian F. Banfield. “The Source and Evolutionary History of a Microbial Contaminant Identified Through Soil Metagenomic Analysis.” *MBio* 8, no. 1 (March 8, 2017): e01969-16. <https://doi.org/10.1128/mBio.01969-16>.

Brown, Christopher T, **Matthew R Olm**, Brian C Thomas, and Jillian F Banfield. “Measurement of Bacterial Replication Rates in Microbial Communities.” *Nature Biotechnology* 34, no. 12 (November 7, 2016): 1256–63. <http://dx.doi.org/10.1038/nbt.3704>.
