---
layout: page
title: Online CV
permalink: /CV/
---
<p align="center">
  <b><big><big>Matthew R. Olm</big></big></b> <br>
  mattolm@berkeley.edu
</p>

### Education

2014-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;PhD, Microbiology; The University of California, Berkeley  
2010-2014
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;BSc, Microbiology; Minor, Computer Science; The University of Pittsburgh  

### Fellowships and honors

2017 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*Genome Research* Cover Image  ([April](https://genome.cshlp.org/content/27/4.cover-expansion))  
2017&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*mBio* Editors' pick ([volume 8, issue 1](http://mbio.asm.org/site/misc/mBioCollection2017.xhtml))  
2014 - 2019 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;National Science Foundation Graduate Research Fellowship  
2012 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Howard Hughes Medical Institute Summer Research Fellowship

### Software

 **dRep** - Python program for rapid comparison and derepliaction of microbial genomes  
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Code](https://github.com/MrOlm/drep) \| [Publication](https://www.nature.com/articles/ismej2017126)  

 **iRep** - Estimating bacteria replication rates based on genome coverage variation  
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Code](https://github.com/christophertbrown/iRep) | [Publication](http://www.nature.com/nbt/journal/v34/n12/abs/nbt.3704.html)  

 *Other programs and scripts* - ([GitHub Account](https://github.com/MrOlm))

### Select scientific presentations

**Academic**
 * 2018 (poster) **Stanford Microbiome Symposium 2018**; Stanford, CA. *Metagenomic analysis reveals bacterial replication rates increase two days prior to necrotizing enterocolitis diagnosis*
 * 2018 (poster) **17th International Symposium on Microbial Ecology**; Leipzig, Germany. *De novo assembly of eukaryotic genomes from metagenomes reveals strain-level overlap between infant and hospital fungal microbiomes*
 * 2017 (oral presentation) **MMG Seminar Series at the University of Pittsburgh**; Pittsburgh, PA. *Using genome-resolved metagenomics to study the developing infant microbiome*
 * 2017 (poster) **Applied Bioinformatics & Public Health Microbiology**; Cambridge, UK. *Genome-resolved metagenomics of infant fecal samples yields thousands of high-quality microbial genomes and reveals patterns of sub-species diversity*
 * 2016 (oral presentation) **Lake Arrowhead Microbial Genomics Conference**; Lake Arrowhead, CA. *Identical Bacterial Populations Colonize Premature Infant Gut, Skin, and Oral Microbiomes and Exhibit Different in situ Growth Rates*
 * 2013 (poster) **American Society for Microbiology General Meeting**; Denver, CO. *Partitioning System of Bacteriophage RedRock Increases Plasmid Maintenance in Mycobacterium smegmatis.*
 * 2012 (poster) **Science 2012**; Pittsburgh, PA. *Study of Putative Partitioning System in Mycobacteriophage RedRock*


**Blog posts**
* [Why genome completeness and contamination estimates are more complicated than you think](https://www.microbe.net/2017/12/13/why-genome-completeness-and-contamination-estimates-are-more-complicated-than-you-think/) _microBE.net_, December 13, 2017  
* [Are these microbes the “same”?](https://www.microbe.net/2017/02/15/are-these-microbes-the-same/) _microBE.net_, February 15, 2017  


**Public engagement** (California Academy of Sciences NightLife; [description](https://www.calacademy.org/nightlife))
* *What's Living in my food? The Good/Bad stuff* (November 17, 2016)
* *History of Beer Microbiology* (July 7, 2016)
* *Who's living in my food? Peeking in on helpful and harmful food microbes* (November 19, 2015)
* *Microbial Fermentation - When, How, and Why?* (September 3, 2015)

### Publications

See [publications](/publications) for full list of publications

1.	**Olm, M.R.**, Bhattacharya, N., Crits-Christoph, A., Firek, B.A., Baker, R., Song, Y.S., Morowitz, M.J., Banfield, J.F., 2019. Necrotizing enterocolitis is preceded by increased gut bacterial replication, Klebsiella, and fimbriae-encoding bacteria that may stimulate TLR4 receptors. **bioRxiv** <http://dx.doi.org/10.1101/558676>

2.	**Olm, M.R.**\*, West, P.T\*., Brooks, B., Firek, B.A., Baker, R., Morowitz, M.J., Banfield, J.F., 2019. Genome-resolved metagenomics of eukaryotic populations during early colonization of premature infants and in hospital rooms. **Microbiome** 7, 26. <https://doi.org/10.1186/s40168-019-0638-1>

3.	**Olm, M.R.**, Brown, C.T., Brooks, B., Banfield, J.F., 2017. dRep: a tool for fast and accurate genomic comparisons that enables improved genome recovery from metagenomes through de-replication. **ISME J.** 11, 2864–2868. <http://dx.doi.org/10.1038/ismej.2017.126>

4.	**Olm, M.R.**, Brown, C.T., Brooks, B., Firek, B., Baker, R., Burstein, D., Soenjoyo, K., Thomas, B.C., Morowitz, M., Banfield, J.F., 2017. Identical bacterial populations colonize premature infant gut, skin, and oral microbiomes and exhibit different in situ growth rates. **Genome Research** 27, 601–612. <http://dx.doi.org/10.1101/gr.213256.116>

5.	**Olm, M.R.**, Butterfield, C.N., Copeland, A., Boles, T.C., Thomas, B.C., Banfield, J.F., 2017. The Source and Evolutionary History of a Microbial Contaminant Identified Through Soil Metagenomic Analysis. **MBio** 8. <http://dx.doi.org/10.1128/mBio.01969-16>

6.	Devoto, A.E., Santini, J.M., **Olm, M.R.**, Anantharaman, K., Munk, P., Tung, J., Archie, E.A., Turnbaugh, P.J., Seed, K.D., Blekhman, R., Aarestrup, F.M., Thomas, B.C., Banfield, J.F., 2019. Megaphages infect Prevotella and variants are widespread in gut microbiomes. **Nature Microbiology** <http://dx.doi.org/10.1038/s41564-018-0338-9>

7.	Matheus Carnevali, P.B., Schulz, F., Castelle, C.J., Kantor, R.S., Shih, P.M., Sharon, I., Santini, J.M., **Olm, M.R.**, Amano, Y., Thomas, B.C., Anantharaman, K., Burstein, D., Becraft, E.D., Stepanauskas, R., Woyke, T., Banfield, J.F., 2019. Hydrogen-based metabolism as an ancestral trait in lineages sibling to the Cyanobacteria. **Nature Communications**. 10, 463. <http://dx.doi.org/10.1038/s41467-018-08246-y>

8.	Rahman, S.F., **Olm, M.R.**, Morowitz, M.J., Banfield, J.F., 2019. Functional potential of bacterial strains in the premature infant gut microbiome is associated with gestational age. **bioRxiv**. <http://dx.doi.org/10.1101/530139>

9.	Sher, Y., **Olm, M.R.**, Raveh-Sadka, T., Brown, C.T., Sher, R., Firek, B., Baker, R., Morowitz, M., Banfield, J., 2018. Shifts in microbial transcription patterns prior to onset of Necrotizing enterocolitis may indicate low oxygen levels in the gut. **bioRxiv**. <http://dx.doi.org/10.1101/466417>

10.	Brooks, B., **Olm, M.R.**, Firek, B.A., Baker, R., Geller-McGrath, D., Reimer, S.R., Soenjoyo, K.R., Yip, J.S., Dahan, D., Thomas, B.C., Morowitz, M.J., Banfield, J.F., 2018. The developing premature infant gut microbiome is a major factor shaping the microbiome of neonatal intensive care unit rooms. **Microbiome** 6, 112. <http://dx.doi.org/10.1186/s40168-018-0493-5>

11.	Brown, C.T., Xiong, W., **Olm, M.R.**, Thomas, B.C., Baker, R., Firek, B., Morowitz, M.J., Hettich, R.L., Banfield, J.F., 2018. Hospitalized Premature Infants Are Colonized by Related Bacterial Strains with Distinct Proteomic Profiles. **MBio** 9. <http://dx.doi.org/10.1128/mBio.00441-18>

12.	Bouma-Gregson, K., **Olm, M.R.**, Probst, A.J., Anantharaman, K., Power, M.E., Banfield, J.F., 2018. Microbial diversity and metabolic potential in cyanotoxin producing cyanobacterial mats throughout a river network. **bioRxiv**. <http://dx.doi.org/10.1101/294421>

13.	Rahman, S.F., **Olm, M.R.**, Morowitz, M.J., Banfield, J.F., 2018. Machine Learning Leveraging Genomes from Metagenomes Identifies Influential Antibiotic Resistance Genes in the Infant Gut Microbiome. **mSystems** 3. <http://dx.doi.org/10.1128/mSystems.00123-17>

14.	Brooks, B., **Olm, M.R.**, Firek, B.A., Baker, R., Thomas, B.C., Morowitz, M.J., Banfield, J.F., 2017. Strain-resolved analysis of hospital rooms and infants reveals overlap between the human and room microbiome. **Nature Communications**. 8, 1814. <http://dx.doi.org/10.1038/s41467-017-02018-w>

15.	Brown, C.T., **Olm, M.R.**, Thomas, B.C., Banfield, J.F., 2016. Measurement of bacterial replication rates in microbial communities. **Nature Biotechnology**. 34, 1256–1263. <http://dx.doi.org/10.1038/nbt.3704>

16.	Dedrick, R.M., Mavrich, T.N., Ng, W.L., Cervantes Reyes, J.C., **Olm, M.R.**, Rush, R.E., Jacobs-Sera, D., Russell, D.A., Hatfull, G.F., 2016. Function, expression, specificity, diversity and incompatibility of actinobacteriophage parABS systems. **Molecular Microbiology**. 101, 625–644. <http://dx.doi.org/10.1111/mmi.13414>

17.	Pope, W.H., Anders, K.R., Baird, M., Bowman, C.A., Boyle, M.M., Broussard, G.W., Chow, T., Clase, K.L., Cooper, S., Cornely, K.A., DeJong, R.J., Delesalle, V.A., Deng, L., Dunbar, D., Edgington, N.P., Ferreira, C.M., Weston Hafer, K., Hartzog, G.A., Hatherill, J.R., Hughes, L.E., Ipapo, K., Krukonis, G.P., Meier, C.G., Monti, D.L., **Olm, M.R.**, Page, S.T., Peebles, C.L., Rinehart, C.A., Rubin, M.R., Russell, D.A., Sanders, E.R., Schoer, M., Shaffer, C.D., Wherley, J., Vazquez, E., Yuan, H., Zhang, D., Cresawn, S.G., Jacobs-Sera, D., Hendrix, R.W., Hatfull, G.F., 2014. Cluster M mycobacteriophages Bongo, PegLeg, and Rey with unusually large repertoires of tRNA isotypes. **Journal of virology**. 88, 2461–2480. <http://dx.doi.org/10.1128/JVI.03363-13>
