---
layout: page
title: Research
permalink: /research/
nav_order: 2
---

I answer fundamental questions about the early microbiome and other microbial communities using innovative bioinformatics methods. 

# The Early Microbiome

{:refdef: style="text-align: left; float: right; margin: 5px;"}
[<img src="{{site.baseurl}}/images/Picture1.png" width="250"/>](https://genome.cshlp.org/content/27/4.cover-expansion)
{: refdef}

Microbes are essential to the health of virtually all plants and animals, yet we have practically no idea where initially colonizing strains come from. A consistent thread throughout my research career has been using deep metagenomic sequencing to study infant strain acquisition, microbiome disorders that impact infants, and how the infant microbiome varies around the globe. I study bacterial, arachael, and eukaryotic microbiome members, as well as bacteriophages and plasmids.
Specific studies include:

* [*Robust variation in infant gut microbiome assembly across a spectrum of lifestyles* (Olm et al., 2022, *Science*)](https://doi.org/10.1126/science.abj2972)

* [*Necrotizing enterocolitis is preceded by increased gut bacterial replication, Klebsiella, and fimbriae-encoding bacteria* (Olm et al., 2019, *Science Advances*)](https://doi.org/10.1126/sciadv.aax5727)

* [*Genome- resolved metagenomics of eukaryotic populations during early colonization of premature infants and in hospital rooms* (Olm et al., 2019, *Microbiome*)](https://doi.org/10.1186/s40168-019-0638-1)

* [*Identical bacterial populations colonize premature infant gut, skin, and oral microbiomes and exhibit different in situ growth rates* (Olm et al., 2017, *Genome Research*)](https://doi.org/10.1101/gr.213256.116)

# Innovative Bioinformatics Methods

{:refdef: style="text-align: left; float: right; margin: 5px;"}
[<img src="{{site.baseurl}}/images/FigureD.png" width="300"/>](https://drep.readthedocs.io/en/latest/overview.html)
{: refdef}

My research strategy of applying computational biology to state-of-the-art -omics techniques can give substantial insight into virtually all microbial environments. I specialize in genome-resolved metagenomic sequencing, an approach that involves sequencing DNA without prior cultivation or amplification, and have published two highly-used programs in the field (dRep and inStrain; 15,000+ downloads each). I also have experience working with transcriptomic, proteonomic, and metabolomic sequencing data, as well as high-dimension immune profiling methods.
Specific studies include:

* [*inStrain profiles population microdiversity from metagenomic data and sensitively detects shared microbial strains* (Olm et al., 2021, *Nature Biotechnology*)](https://doi.org/10.1038/s41587-020-00797-0)

* [*dRep: a tool for fast and accurate genomic comparisons that enables improved genome recovery from metagenomes through de-replication* (Olm et al., 2017, *The ISME Journal*)](https://doi.org/10.1038/ismej.2017.126)

* [*Measurement of bacterial replication rates in microbial communities* (Brown et al., 2017, *Nature Biotechnology*)](https://doi.org/10.1038/nbt.3704)

# Fundamental Questions of Biology

{:refdef: style="text-align: left; float: right; margin: 5px;"}
[<img src="{{site.baseurl}}/images/msystems.00731-19-f0003.jpeg" width="300"/>](https://doi.org/10.1128/mSystems.00731-19)
{: refdef}

The invention of the Petri Dish allowed scientists to cultivate and study individual microbial species for the first time, resulting in a 30-year period of discovery known as the “Golden Age of Microbiology”. The invention of genome-resolved metagenomics in 2004 allowed scientists to study microbial species without the need for cultivation for the first time, leading to an complete revolution of our understanding of life in general. I believe we are in the midst of a second “Golden Age of Microbiology”, ushered in by -omics techniques and computational biology, and enjoy using these techniques to pursue fundamental biologist questions.
Specific studies include:

* [*Consistent Metagenome-Derived Metrics Verify and Delineate Bacterial Species* (Olm et al., 2020, *mSystems*)](https://doi.org/10.1128/mSystems.00731-19)

* [*Clades of huge phages from across Earth’s ecosystems* (Al-Shayeb et al., 2020, *Nature*)](https://doi.org/10.1038/s41586-020-2007-4)
