<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1"><!-- Begin Jekyll SEO tag v2.4.0 -->
<title>Online CV | Matthew R. Olm</title>
<meta name="generator" content="Jekyll v3.7.3" />
<meta property="og:title" content="Online CV" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Matt Olm - PhD Candidate in Jill Banfield’s lab at UC-Berkeley." />
<meta property="og:description" content="Matt Olm - PhD Candidate in Jill Banfield’s lab at UC-Berkeley." />
<link rel="canonical" href="http://localhost:4000/CV/" />
<meta property="og:url" content="http://localhost:4000/CV/" />
<meta property="og:site_name" content="Matthew R. Olm" />
<script type="application/ld+json">
{"@type":"WebPage","url":"http://localhost:4000/CV/","description":"Matt Olm - PhD Candidate in Jill Banfield’s lab at UC-Berkeley.","headline":"Online CV","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<link rel="stylesheet" href="/assets/main.css"><link type="application/atom+xml" rel="alternate" href="http://localhost:4000/feed.xml" title="Matthew R. Olm" /></head>
<body><header class="site-header" role="banner">

  <div class="wrapper"><a class="site-title" rel="author" href="/">Matthew R. Olm</a><nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
              <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
              <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger"><a class="page-link" href="/CV/">Online CV</a><a class="page-link" href="/">About</a><a class="page-link" href="/publications/">Publications</a><a class="page-link" href="/publications2/">Publications2</a></div>
      </nav></div>
</header>
<main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post">

  <header class="post-header">
    <h1 class="post-title">Online CV</h1>
  </header>

  <div class="post-content">
    <p align="center">
  <b><big><big>Matthew R. Olm</big></big></b> <br />
  mattolm@berkeley.edu
</p>

<h3 id="education">Education</h3>

<p>2014-         
      PhD, Microbiology; The University of California, Berkeley<br />
2010-2014
      BSc, Microbiology; Minor, Computer Science; The University of Pittsburgh</p>

<h3 id="fellowships-and-honors">Fellowships and honors</h3>

<p>2017                   <em>Genome Research</em> Cover Image  (<a href="https://genome.cshlp.org/content/27/4.cover-expansion">April</a>)<br />
2017                   <em>mBio</em> Editors’ pick (<a href="http://mbio.asm.org/site/misc/mBioCollection2017.xhtml">volume 8, issue 1</a>)<br />
2014 - 2019      National Science Foundation Graduate Research Fellowship<br />
2012                   Howard Hughes Medical Institute Summer Research Fellowship</p>

<h3 id="software">Software</h3>

<p><strong>dRep</strong> - Python program for rapid comparison and derepliaction of microbial genomes<br />
       <a href="https://github.com/MrOlm/drep">Code</a> | <a href="https://www.nature.com/articles/ismej2017126">Publication</a></p>

<p><strong>iRep</strong> - Estimating bacteria replication rates based on genome coverage variation<br />
       <a href="https://github.com/christophertbrown/iRep">Code</a> | <a href="http://www.nature.com/nbt/journal/v34/n12/abs/nbt.3704.html">Publication</a></p>

<p><em>Other programs and scripts</em> - (<a href="https://github.com/MrOlm">GitHub Account</a>)</p>

<h3 id="select-scientific-presentations">Select scientific presentations</h3>

<p><strong>Academic</strong></p>
<ul>
  <li>2018 (poster) <strong>Stanford Microbiome Symposium 2018</strong>; Stanford, CA. <em>Metagenomic analysis reveals bacterial replication rates increase two days prior to necrotizing enterocolitis diagnosis</em></li>
  <li>2018 (poster) <strong>17th International Symposium on Microbial Ecology</strong>; Leipzig, Germany. <em>De novo assembly of eukaryotic genomes from metagenomes reveals strain-level overlap between infant and hospital fungal microbiomes</em></li>
  <li>2017 (oral presentation) <strong>MMG Seminar Series at the University of Pittsburgh</strong>; Pittsburgh, PA. <em>Using genome-resolved metagenomics to study the developing infant microbiome</em></li>
  <li>2017 (poster) <strong>Applied Bioinformatics &amp; Public Health Microbiology</strong>; Cambridge, UK. <em>Genome-resolved metagenomics of infant fecal samples yields thousands of high-quality microbial genomes and reveals patterns of sub-species diversity</em></li>
  <li>2016 (oral presentation) <strong>Lake Arrowhead Microbial Genomics Conference</strong>; Lake Arrowhead, CA. <em>Identical Bacterial Populations Colonize Premature Infant Gut, Skin, and Oral Microbiomes and Exhibit Different in situ Growth Rates</em></li>
  <li>2013 (poster) <strong>American Society for Microbiology General Meeting</strong>; Denver, CO. <em>Partitioning System of Bacteriophage RedRock Increases Plasmid Maintenance in Mycobacterium smegmatis.</em></li>
  <li>2012 (poster) <strong>Science 2012</strong>; Pittsburgh, PA. <em>Study of Putative Partitioning System in Mycobacteriophage RedRock</em></li>
</ul>

<p><strong>Blog posts</strong></p>
<ul>
  <li><a href="https://www.microbe.net/2017/12/13/why-genome-completeness-and-contamination-estimates-are-more-complicated-than-you-think/">Why genome completeness and contamination estimates are more complicated than you think</a> <em>microBE.net</em>, December 13, 2017</li>
  <li><a href="https://www.microbe.net/2017/02/15/are-these-microbes-the-same/">Are these microbes the “same”?</a> <em>microBE.net</em>, February 15, 2017</li>
</ul>

<p><strong>Public engagement</strong> (California Academy of Sciences NightLife; <a href="https://www.calacademy.org/nightlife">description</a>)</p>
<ul>
  <li><em>What’s Living in my food? The Good/Bad stuff</em> (November 17, 2016)</li>
  <li><em>History of Beer Microbiology</em> (July 7, 2016)</li>
  <li><em>Who’s living in my food? Peeking in on helpful and harmful food microbes</em> (November 19, 2015)</li>
  <li><em>Microbial Fermentation - When, How, and Why?</em> (September 3, 2015)</li>
</ul>

<h3 id="first-and-second-author-publications">First and Second Author Publications</h3>

<p>See <a href="/publications">publications</a> for full list of publications</p>

<p><strong>Olm, Matthew R</strong>, Nicholas Bhattacharya, Alexander Crits-Christoph, Brian A. Firek, Robyn Baker, Yun S. Song, Michael J. Morowitz, and Jillian F. Banfield. “Necrotizing Enterocolitis Is Preceded by Increased Gut Bacterial Replication, Klebsiella, and Fimbriae-Encoding Bacteria That May Stimulate TLR4 Receptors.” <em>bioRxiv</em> (February 22, 2019). <a href="https://doi.org/10.1101/558676">https://doi.org/10.1101/558676</a></p>

<p><strong>Olm, Matthew R</strong>*, Patrick T West*, Brandon Brooks, Brian A. Firek, Robyn Baker, Michael J. Morowitz, and Jillian F. Banfield. “Genome-Resolved Metagenomics of Eukaryotic Populations during Early Colonization of Premature Infants and in Hospital Rooms.” <em>Microbiome</em> 7 (1): 26 (February 15, 2019). <a href="https://doi.org/10.1186/s40168-019-0638-1">https://doi.org/10.1186/s40168-019-0638-1</a></p>

<p>Rahman, Sumayah F., <strong>Matthew R. Olm</strong>, Michael J. Morowitz, and Jillian F. Banfield. 2019. “Functional Potential of Bacterial Strains in the Premature Infant Gut Microbiome Is Associated with Gestational Age.” <em>bioRxiv</em> (January 24, 2019). <a href="https://doi.org/10.1101/530139">https://doi.org/10.1101/530139</a></p>

<p>Sher, Yonatan, <strong>Matthew R Olm</strong>, Tali Raveh-Sadka, Christopher T Brown, Ruth Sher, Brian Firek, Robyn Baker, Michael Morowitz, Jill Banfield. “Shifts in microbial transcription patterns prior to onset of Necrotizing enterocolitis may indicate low oxygen levels in the gut.” <em>BioRxiv</em> (November 9, 2018). <a href="https://doi.org/10.1101/466417">https://doi.org/10.1101/466417</a></p>

<p>Brooks, Brandon, <strong>Matthew R Olm</strong>, Brian A. Firek, Robyn Baker, David Geller-McGrath, Sophia R. Reimer, Karina R. Soenjoyo, et al. “The Developing Premature Infant Gut Microbiome Is a Major Factor Shaping the Microbiome of Neonatal Intensive Care Unit Rooms.” <em>Microbiome</em> 6:112 (June 20, 2018). <a href="https://doi.org/10.1186/s40168-018-0493-5">https://doi.org/10.1186/s40168-018-0493-5</a>.</p>

<p>Bouma-Gregson, Keith, <strong>Matthew R Olm</strong>, Alexander J Probst, Karthik Anantharaman, Mary E Power, and Jillian F Banfield. “Microbial Diversity and Metabolic Potential in Cyanotoxin Producing Cyanobacterial Mats throughout a River Network.” <em>BioRxiv</em> (In press at <em>The ISME Journal</em>) (April 3, 2018). <a href="https://doi.org/10.1101/294421">https://doi.org/10.1101/294421</a>.</p>

<p>Rahman, Sumayah F., <strong>Matthew R. Olm</strong>, Michael J. Morowitz, and Jillian F. Banfield. “Machine Learning Leveraging Genomes from Metagenomes Identifies Influential Antibiotic Resistance Genes in the Infant Gut Microbiome.” <em>MSystems</em> 3, no. 1 (February 27, 2018). <a href="https://doi.org/10.1128/mSystems.00123-17">https://doi.org/10.1128/mSystems.00123-17</a>.</p>

<p>Brooks, Brandon, <strong>Matthew R. Olm</strong>, Brian A. Firek, Robyn Baker, Brian C. Thomas, Michael J. Morowitz, and Jillian F. Banfield. “Strain-Resolved Analysis of Hospital Rooms and Infants Reveals Overlap between the Human and Room Microbiome.” <em>Nature Communications</em> 8, no. 1 (November 27, 2017): 1814. <a href="https://doi.org/10.1038/s41467-017-02018-w">https://doi.org/10.1038/s41467-017-02018-w</a>.</p>

<p><strong>Olm, Matthew R</strong>, Christopher T Brown, Brandon Brooks, and Jillian F Banfield. “DRep: A Tool for Fast and Accurate Genomic Comparisons That Enables Improved Genome Recovery from Metagenomes through de-Replication.” <em>The ISME Journal</em> 11 (July 25, 2017): 2864. <a href="http://dx.doi.org/10.1038/ismej.2017.126">http://dx.doi.org/10.1038/ismej.2017.126</a>.</p>

<p><strong>Olm, Matthew R.</strong>, Christopher T. Brown, Brandon Brooks, Brian Firek, Robyn Baker, David Burstein, Karina Soenjoyo, Brian C. Thomas, Michael Morowitz, and Jillian F. Banfield. “Identical Bacterial Populations Colonize Premature Infant Gut, Skin, and Oral Microbiomes and Exhibit Different in Situ Growth Rates.” <em>Genome Research</em> 27, no. 4 (April 1, 2017): 601–12. <a href="https://doi.org/10.1101/gr.213256.116">https://doi.org/10.1101/gr.213256.116</a>.</p>

<p><strong>Olm, Matthew R.</strong>, Cristina N. Butterfield, Alex Copeland, T. Christian Boles, Brian C. Thomas, and Jillian F. Banfield. “The Source and Evolutionary History of a Microbial Contaminant Identified Through Soil Metagenomic Analysis.” <em>MBio</em> 8, no. 1 (March 8, 2017): e01969-16. <a href="https://doi.org/10.1128/mBio.01969-16">https://doi.org/10.1128/mBio.01969-16</a>.</p>

<p>Brown, Christopher T, <strong>Matthew R Olm</strong>, Brian C Thomas, and Jillian F Banfield. “Measurement of Bacterial Replication Rates in Microbial Communities.” <em>Nature Biotechnology</em> 34, no. 12 (November 7, 2016): 1256–63. <a href="http://dx.doi.org/10.1038/nbt.3704">http://dx.doi.org/10.1038/nbt.3704</a>.</p>

  </div>

</article>

      </div>
    </main><footer class="site-footer h-card">
  <data class="u-url" href="/"></data>

  <div class="wrapper">

    <h2 class="footer-heading">Matthew R. Olm</h2>

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li class="p-name">Matthew R. Olm</li><li><a class="u-email" href="mailto:mattolm@berkeley.edu">mattolm@berkeley.edu</a></li></ul>
      </div>

      <div class="footer-col footer-col-2"><ul class="social-media-list"><li><a href="https://github.com/mrolm"><svg class="svg-icon"><use xlink:href="/assets/minima-social-icons.svg#github"></use></svg> <span class="username">mrolm</span></a></li><li><a href="https://www.twitter.com/mattagenolmics"><svg class="svg-icon"><use xlink:href="/assets/minima-social-icons.svg#twitter"></use></svg> <span class="username">mattagenolmics</span></a></li></ul>
</div>

      <div class="footer-col footer-col-3">
        <p>Matt Olm - PhD Candidate in Jill Banfield&#39;s lab at UC-Berkeley.
</p>
      </div>
    </div>

  </div>

</footer>
</body>

</html>
