---
layout: post
title: Publications
permalink: /publications/
---
<script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>


<a href="https://scholar.google.com/citations?user=BoDUkpMAAAAJ&hl"><big>Link to google scholar</big></a>
</br>
</br>
<p>
<body> <i><a href="https://doi.org/10.1126/science.abj2972"> Robust variation in infant gut microbiome assembly across a spectrum of lifestyles</a></i> - <b>Science</b> (2022)<br/><b>Olm M.R.</b>,  Dahan D.,  Carter M.M.,  Merrill B.D.,  Yu F.B.,  Jain S.,  Meng X.,  Tripathi S.,  Wastyk H.,  Neff N. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1126/science.abj2972"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1126/science.abj2972 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="http://biorxiv.org/content/early/2022/03/31/2022.03.30.486478.abstract"> Ultra-deep Sequencing of Hadza Hunter-Gatherers Recovers Vanishing Microbes</a></i> - <b>bioRxiv</b> (2022)<br/>Merrill B.D.,  Carter M.M.,  <b>Olm M.R.</b>,  Dahan D.,  Tripathi S.,  Spencer S.P.,  Yu B.,  Jain S.,  Neff N.,  Jha A.R.,  Sonnenburg E.D.,  Sonnenburg J.L. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1101/2022.03.30.486478"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1101/2022.03.30.486478 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="http://biorxiv.org/content/early/2022/01/17/2022.01.16.476537.abstract"> Using strain-resolved analysis to identify contamination in metagenomics data</a></i> - <b>bioRxiv</b> (2022)<br/>Lou Y.C.,  Hoff J.,  <b>Olm M.R.</b>,  West-Roberts J.,  Diamond S.,  Firek B.A.,  Morowitz M.J.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1101/2022.01.16.476537"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1101/2022.01.16.476537 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://www.nature.com/articles/s41564-021-01036-1"> Microbiome assembly in The Gambia</a></i> - <b>Nature Microbiology</b> (2021)<br/>Carter M.M.,  <b>Olm M.R.</b>,  Sonnenburg E.D. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s41564-021-01036-1"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1038/s41564-021-01036-1 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://doi.org/10.1038/s41591-021-01552-x"> Reporting guidelines for human microbiome research: the STORMS checklist</a></i> - <b>Nature Medicine</b> (2021)<br/>Mirzayi C.,  Renson A.,  Furlanello C.,  Sansone S.,  Zohra F.,  Elsafoury S.,  Geistlinger L.,  Kasselman L.J.,  Eckenrode K.,  van d.W.J.,  Loughman A.,  Marques F.Z.,  MacIntyre D.A.,  Arumugam M.,  Azhar R.,  Beghini F.,  Bergstrom K.,  Bhatt A.,  Bisanz J.E.,  Braun J.,  Bravo H.C.,  Buck G.A.,  Bushman F.,  Casero D.,  Clarke G.,  Collado M.C.,  Cotter P.D.,  Cryan J.F.,  Demmer R.T.,  Devkota S.,  Elinav E.,  Escobar J.S.,  Fettweis J.,  Finn R.D.,  Fodor A.A.,  Forslund S.,  Franke A.,  Furlanello C.,  Gilbert J.,  Grice E.,  Haibe-Kains B.,  Handley S.,  Herd P.,  Holmes S.,  Jacobs J.P.,  Karstens L.,  Knight R.,  Knights D.,  Koren O.,  Kwon D.S.,  Langille M.,  Lindsay B.,  McGovern D.,  McHardy A.C.,  McWeeney S.,  Mueller N.T.,  Nezi L.,  <b>Olm M.</b>,  Palm N.,  Pasolli E.,  Raes J.,  Redinbo M.R.,  Rühlemann M.,  Balfour S.R.,  Schloss P.D.,  Schriml L.,  Segal E.,  Shardell M.,  Sharpton T.,  Smirnova E.,  Sokol H.,  Sonnenburg J.L.,  Srinivasan S.,  Thingholm L.B.,  Turnbaugh P.J.,  Upadhyay V.,  Walls R.L.,  Wilmes P.,  Yamada T.,  Zeller G.,  Zhang M.,  Zhao N.,  Zhao L.,  Bao W.,  Culhane A.,  Devanarayan V.,  Dopazo J.,  Fan X.,  Fischer M.,  Jones W.,  Kusko R.,  Mason C.E.,  Mercer T.R.,  Sansone S.,  Scherer A.,  Shi L.,  Thakkar S.,  Tong W.,  Wolfinger R.,  Hunter C.,  Segata N.,  Huttenhower C.,  Dowd J.B.,  Jones H.E.,  Waldron L.,  undefined ,  undefined  </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s41591-021-01552-x"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1038/s41591-021-01552-x data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://onlinelibrary.wiley.com/doi/10.1111/mec.16208"> <i>Microcoleus</i> (Cyanobacteria) form watershed‐wide populations without strong gradients in population structure</a></i> - <b>Molecular Ecology</b> (2021)<br/>Bouma‐Gregson K.,  Crits‐Christoph A.,  <b>Olm M.R.</b>,  Power M.E.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1111/mec.16208"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1111/mec.16208 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://www.sciencedirect.com/science/article/pii/S2666379121002512"> Infant gut strain persistence is associated with maternal origin, phylogeny, and traits including surface adhesion and iron acquisition</a></i> - <b>Cell Reports Medicine</b> (2021)<br/>Lou Y.C.,  <b>Olm M.R.</b>,  Diamond S.,  Crits-Christoph A.,  Firek B.A.,  Baker R.,  Morowitz M.J.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1016/j.xcrm.2021.100393"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1016/j.xcrm.2021.100393 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://doi.org/10.1186/s40168-021-01085-y"> Genetic and behavioral adaptation of Candida parapsilosis to the microbiome of hospitalized infants revealed by in situ genomics, transcriptomics, and proteomics</a></i> - <b>Microbiome</b> (2021)<br/>West P.T.,  Peters S.L.,  <b>Olm M.R.</b>,  Yu F.B.,  Gause H.,  Lou Y.C.,  Firek B.A.,  Baker R.,  Johnson A.D.,  Morowitz M.J.,  Hettich R.L.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1186/s40168-021-01085-y"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1186/s40168-021-01085-y data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="http://www.nature.com/articles/s41587-020-00797-0"> inStrain profiles population microdiversity from metagenomic data and sensitively detects shared microbial strains</a></i> - <b>Nature Biotechnology</b> (2021)<br/><b>Olm M.R.</b>,  Crits-Christoph A.,  Bouma-Gregson K.,  Firek B.A.,  Morowitz M.J.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s41587-020-00797-0"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1038/s41587-020-00797-0 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://doi.org/10.1186/s40168-020-00957-z"> Meanders as a scaling motif for understanding of floodplain soil microbiome and biogeochemical potential at the watershed scale</a></i> - <b>Microbiome</b> (2021)<br/>Carnevali P.B.M.,  Lavy A.,  Thomas A.D.,  Crits-Christoph A.,  Diamond S.,  Méheust R.,  <b>Olm M.R.</b>,  Sharrar A.,  Lei S.,  Dong W. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="https://doi.org/10.1186/s40168-020-00957-z"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=https://doi.org/10.1186/s40168-020-00957-z data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://doi.org/10.1038/d41586-021-01266-7"> Ancient human faeces reveal gut microbes of the past</a></i> - <b>Nature</b> (2021)<br/><b>Olm M.R.</b>,  Sonnenburg J.L. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="https://doi.org/10.1038/d41586-021-01266-7"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=https://doi.org/10.1038/d41586-021-01266-7 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="http://mbio.asm.org/content/12/1/e02703-20.abstract"> Genome Sequencing of Sewage Detects Regionally Prevalent SARS-CoV-2 Variants</a></i> - <b>mBio</b> (2021)<br/>Crits-Christoph A.,  Kantor R.S.,  <b>Olm M.R.</b>,  Whitney O.N.,  Al-Shayeb B.,  Lou Y.C.,  Flamholz A.,  Kennedy L.C.,  Greenwald H.,  Hinkle A.,  Hetzel J.,  Spitzer S.,  Koble J.,  Tan A.,  Hyde F.,  Schroth G.,  Kuersten S.,  Banfield J.F.,  Nelson K.L. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1128/mBio.02703-20"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1128/mBio.02703-20 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="http://genome.cshlp.org/lookup/doi/10.1101/gr.268169.120"> Transporter genes in biosynthetic gene clusters predict metabolite characteristics and siderophore activity</a></i> - <b>Genome Research</b> (2021)<br/>Crits-Christoph A.,  Bhattacharya N.,  <b>Olm M.R.</b>,  Song Y.S.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1101/gr.268169.120"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1101/gr.268169.120 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://doi.org/10.1038/s41396-020-0655-x"> Soil bacterial populations are shaped by recombination and gene-specific selection across a grassland meadow</a></i> - <b>The ISME Journal</b> (2020)<br/>Crits-Christoph A.,  <b>Olm M.R.</b>,  Diamond S.,  Bouma-Gregson K.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s41396-020-0655-x"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1038/s41396-020-0655-x data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://doi.org/10.1371/journal.pone.0229537"> Combined analysis of microbial metagenomic and metatranscriptomic sequencing data to assess in situ physiological conditions in the premature infant gut</a></i> - <b>Plos one</b> (2020)<br/>Sher Y.,  <b>Olm M.R.</b>,  Raveh-Sadka T.,  Brown C.T.,  Sher R.,  Firek B.,  Baker R.,  Morowitz M.J.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1371/journal.pone.0229537"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1371/journal.pone.0229537 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://msystems.asm.org/content/5/1/e00731-19.abstract"> Consistent Metagenome-Derived Metrics Verify and Delineate Bacterial Species Boundaries</a></i> - <b>mSystems</b> (2020)<br/><b>Olm M.R.</b>,  Crits-Christoph A.,  Diamond S.,  Lavy A.,  Carnevali P.B.M.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1128/mSystems.00731-19"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1128/mSystems.00731-19 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://www.nature.com/articles/s41586-020-2007-4"> Clades of huge phages from across Earth’s ecosystems</a></i> - <b>Nature</b> (2020)<br/>Al-Shayeb B.,  Sachdeva R.,  Chen L.,  Ward F.,  Munk P.,  Devoto A.,  Castelle C.J.,  <b>Olm M.R.</b>,  Bouma-Gregson K.,  Amano Y.,  He C.,  Méheust R.,  Brooks B.,  Thomas A.,  Lavy A.,  Matheus-Carnevali P.,  Sun C.,  Goltsman D.S.A.,  Borton M.A.,  Sharrar A.,  Jaffe A.L.,  Nelson T.C.,  Kantor R.,  Keren R.,  Lane K.R.,  Farag I.F.,  Lei S.,  Finstad K.,  Amundson R.,  Anantharaman K.,  Zhou J.,  Probst A.J.,  Power M.E.,  Tringe S.G.,  Li W.,  Wrighton K.,  Harrison S.,  Morowitz M.,  Relman D.A.,  Doudna J.A.,  Lehours A.,  Warren L.,  Cate J.H.D.,  Santini J.M.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s41586-020-2007-4"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1038/s41586-020-2007-4 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://advances.sciencemag.org/content/5/12/eaax5727"> Necrotizing enterocolitis is preceded by increased gut bacterial replication, Klebsiella, and fimbriae-encoding bacteria</a></i> - <b>Science advances</b> (2019)<br/><b>Olm M.R.</b>,  Bhattacharya N.,  Crits-Christoph A.,  Firek B.A.,  Baker R.,  Song Y.S.,  Morowitz M.J.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1126/sciadv.aax5727"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1126/sciadv.aax5727 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://www.nature.com/articles/s41564-018-0338-9"> Megaphages infect Prevotella and variants are widespread in gut microbiomes</a></i> - <b>Nature Microbiology</b> (2019)<br/>Devoto A.E.,  Santini J.M.,  <b>Olm M.R.</b>,  Anantharaman K.,  Munk P.,  Tung J.,  Archie E.A.,  Turnbaugh P.J.,  Seed K.D.,  Blekhman R.,  Aarestrup F.M.,  Thomas B.C.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s41564-018-0338-9"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1038/s41564-018-0338-9 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://www.nature.com/articles/s41396-019-0374-3"> Impacts of microbial assemblage and environmental conditions on the distribution of anatoxin-a producing cyanobacteria within a river network</a></i> - <b>The ISME Journal</b> (2019)<br/>Bouma-Gregson K.,  <b>Olm M.R.</b>,  Probst A.J.,  Anantharaman K.,  Power M.E.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s41396-019-0374-3"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1038/s41396-019-0374-3 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://doi.org/10.1186/s40168-019-0638-1"> Genome-resolved metagenomics of eukaryotic populations during early colonization of premature infants and in hospital rooms</a></i> - <b>Microbiome</b> (2019)<br/><b>Olm M.R.</b>,  West P.T.,  Brooks B.,  Firek B.A.,  Baker R.,  Morowitz M.J.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1186/s40168-019-0638-1"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1186/s40168-019-0638-1 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://www.nature.com/articles/s41467-018-08246-y"> Hydrogen-based metabolism as an ancestral trait in lineages sibling to the Cyanobacteria</a></i> - <b>Nature Communications</b> (2019)<br/>Carnevali P.B.M.,  Schulz F.,  Castelle C.J.,  Kantor R.S.,  Shih P.M.,  Sharon I.,  Santini J.M.,  <b>Olm M.R.</b>,  Amano Y.,  Thomas B.C.,  Anantharaman K.,  Burstein D.,  Becraft E.D.,  Stepanauskas R.,  Woyke T.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s41467-018-08246-y"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1038/s41467-018-08246-y data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://www.biorxiv.org/content/10.1101/530139v1"> Functional potential of bacterial strains in the premature infant gut microbiome is associated with gestational age</a></i> - <b>bioRxiv</b> (2019)<br/>Rahman S.F.,  <b>Olm M.R.</b>,  Morowitz M.J.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1101/530139"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1101/530139 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://doi.org/10.1186/s40168-018-0493-5"> The developing premature infant gut microbiome is a major factor shaping the microbiome of neonatal intensive care unit rooms</a></i> - <b>Microbiome</b> (2018)<br/>Brooks B.,  <b>Olm M.R.</b>,  Firek B.A.,  Baker R.,  Geller-McGrath D.,  Reimer S.R.,  Soenjoyo K.R.,  Yip J.S.,  Dahan D.,  Thomas B.C.,  Morowitz M.J.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1186/s40168-018-0493-5"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1186/s40168-018-0493-5 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://mbio.asm.org/content/9/2/e00441-18"> Hospitalized Premature Infants Are Colonized by Related Bacterial Strains with Distinct Proteomic Profiles</a></i> - <b>mBio</b> (2018)<br/>Brown C.T.,  Xiong W.,  <b>Olm M.R.</b>,  Thomas B.C.,  Baker R.,  Firek B.,  Morowitz M.J.,  Hettich R.L.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1128/mBio.00441-18"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1128/mBio.00441-18 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://msystems.asm.org/content/3/1/e00123-17"> Machine Learning Leveraging Genomes from Metagenomes Identifies Influential Antibiotic Resistance Genes in the Infant Gut Microbiome</a></i> - <b>mSystems</b> (2018)<br/>Rahman S.F.,  <b>Olm M.R.</b>,  Morowitz M.J.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1128/mSystems.00123-17"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1128/mSystems.00123-17 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://www.nature.com/articles/s41467-017-02018-w"> Strain-resolved analysis of hospital rooms and infants reveals overlap between the human and room microbiome</a></i> - <b>Nature Communications</b> (2017)<br/>Brooks B.,  <b>Olm M.R.</b>,  Firek B.A.,  Baker R.,  Thomas B.C.,  Morowitz M.J.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s41467-017-02018-w"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1038/s41467-017-02018-w data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://www.nature.com/articles/nbt.3704"> Measurement of bacterial replication rates in microbial communities</a></i> - <b>Nature Biotechnology</b> (2017)<br/>Brown C.T.,  <b>Olm M.R.</b>,  Thomas B.C.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/nbt.3704"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1038/nbt.3704 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://www.nature.com/articles/ismej2017126"> dRep: a tool for fast and accurate genomic comparisons that enables improved genome recovery from metagenomes through de-replication</a></i> - <b>The ISME Journal</b> (2017)<br/><b>Olm M.R.</b>,  Brown C.T.,  Brooks B.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/ismej.2017.126"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1038/ismej.2017.126 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="http://genome.cshlp.org/content/27/4/601"> Identical bacterial populations colonize premature infant gut, skin, and oral microbiomes and exhibit different in situ growth rates</a></i> - <b>Genome Research</b> (2017)<br/><b>Olm M.R.</b>,  Brown C.T.,  Brooks B.,  Firek B.,  Baker R.,  Burstein D.,  Soenjoyo K.,  Thomas B.C.,  Morowitz M.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1101/gr.213256.116"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1101/gr.213256.116 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://mbio.asm.org/content/8/1/e01969-16"> The Source and Evolutionary History of a Microbial Contaminant Identified Through Soil Metagenomic Analysis</a></i> - <b>mBio</b> (2017)<br/><b>Olm M.R.</b>,  Butterfield C.N.,  Copeland A.,  Boles T.C.,  Thomas B.C.,  Banfield J.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1128/mBio.01969-16"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1128/mBio.01969-16 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/mmi.13414"> Function, expression, specificity, diversity and incompatibility of actinobacteriophage parABS systems</a></i> - <b>Molecular Microbiology</b> (2016)<br/>Dedrick R.M.,  Mavrich T.N.,  Ng W.L.,  Reyes J.C.C.,  <b>Olm M.R.</b>,  Rush R.E.,  Jacobs‐Sera D.,  Russell D.A.,  Hatfull G.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1111/mmi.13414"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1111/mmi.13414 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
<p>
<body> <i><a href="https://jvi.asm.org/content/88/5/2461"> Cluster M Mycobacteriophages Bongo, PegLeg, and Rey with Unusually Large Repertoires of tRNA Isotypes</a></i> - <b>Journal of Virology</b> (2014)<br/>Pope W.H.,  Anders K.R.,  Baird M.,  Bowman C.A.,  Boyle M.M.,  Broussard G.W.,  Chow T.,  Clase K.L.,  Cooper S.,  Cornely K.A.,  DeJong R.J.,  Delesalle V.A.,  Deng L.,  Dunbar D.,  Edgington N.P.,  Ferreira C.M.,  Hafer K.W.,  Hartzog G.A.,  Hatherill J.R.,  Hughes L.E.,  Ipapo K.,  Krukonis G.P.,  Meier C.G.,  Monti D.L.,  <b>Olm M.R.</b>,  Page S.T.,  Peebles C.L.,  Rinehart C.A.,  Rubin M.R.,  Russell D.A.,  Sanders E.R.,  Schoer M.,  Shaffer C.D.,  Wherley J.,  Vazquez E.,  Yuan H.,  Zhang D.,  Cresawn S.G.,  Jacobs-Sera D.,  Hendrix R.W.,  Hatfull G.F. </body>
<span style="display:block">
<span style="display:inline" class='altmetric-embed' data-badge-type='donut' data-doi="10.1128/JVI.03363-13"></span>
<span style="display:inline" class="__dimensions_badge_embed__" data-doi=10.1128/JVI.03363-13 data-style="small_circle" data-hide-zero-citations="true"></span>
</span>

</p>

<hr/>
